apps/backoffice-v2/src/common/utils/fetcher/fetcher.ts

```diff
@@ -56,7 +56,7 @@ export const fetcher: IFetcher = async ({
 
   const [data, jsonError] = isBlob
     ? await handlePromise(res.blob())
-    : res.headers.get('content-length') > '0'
+    : !res.headers.get('content-length') || res.headers.get('content-length') > '0'
```

Comment:
This is a bug fix - before if headers was an empty object undefined would always be returned as data.